src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -32,10 +32,16 @@ sylius_fixtures:
                         countries:
                             - "US"
                         zones:
-                            US:
+                            US-TAX:
                                 name: "United States of America"
                                 countries:
                                     - "US"
+                                scope: "tax"
+                            US-SHIPPING:
+                                name: "United States of America"
+                                countries:
+                                    - "US"
+                                scope: "shipping"
```

Comment:
I would leave a single zone with scope `all` for fixtures since we with current data set we do not benefit from having them separate.